dev-deploy-templates.sh

```diff
@@ -12,8 +12,8 @@ S3_BUCKET_URL=s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/scipoolprod
 # list the templates you're testing
 templates=(
   ec2/sc-portfolio-ec2-development.yaml
-  ec2/sc-product-ec2-linux-jumpcloud-notebook-development.yaml
-  ec2/sc-ec2-linux-jumpcloud-notebook-development.yaml
+  ec2/sc-product-ec2-linux-jumpcloud-workflows-development.yaml
```

Comment:
why is this a replacement instead of an addition?

Reply:
I've been replacing with whatever dev templates I'm working on at the moment. I could add all dev templates. But if more than one of us wants to work on it at once and I add all dev templates, we'll override each other. It's not an ideal solution, and I haven't put any time into making it more robust.